core/src/main/java/io/github/tcdl/msb/support/Utils.java

```diff
@@ -56,8 +58,8 @@ public static String toJson(Object object, ObjectMapper objectMapper) {
         try {
             return objectMapper.writeValueAsString(object);
         } catch (JsonProcessingException e) {
-            LOG.error("Failed parse to JSON from: " + object, e);
-            throw new JsonConversionException(e.getMessage());
+            LOG.error("Failed to parse to JSON object: [{}] ", object);
```

Comment:
I know that you didn't change this code but "log & throw" is considered evil: http://stackoverflow.com/questions/6639963/why-is-log-and-throw-condsidered-an-anti-pattern

In our case we'll end up logging the same parsing issue from multiple catch blocks in different methods. Could we get rid of it?